docs/use_local_hyperledger.md

```diff
@@ -0,0 +1,102 @@
+#Use Local Hyperledger Network:
+
+### Creating a Local Hyperledger Network
+There is a convenient docker-compose image that will get you a network very quickly.
```

Comment:
It isn't a docker-compose 'image'.  I'd reword this to something like, "It's easy to stand up a local hyperledger network using docker compose."